front/app/containers/IdeasShow/components/Container.tsx

```diff
@@ -114,39 +103,20 @@ const Container2 = ({
         enter={true}
         exit={false}
       >
-        <main id="e2e-idea-show">
-          <Container
-            className={`loaded ${className || ''}`}
-            ref={handleContainerRef}
-          >
-            {children}
-          </Container>
-        </main>
+        <Container
+          className={`loaded ${className || ''}`}
+          ref={handleContainerRef}
+        >
+          {children}
+        </Container>
       </CSSTransition>
-      <Modal
-        opened={!!newIdeaId}
-        close={closeIdeaSocialSharingModal}
-        hasSkipButton={true}
-        skipText={<>{formatMessage(messages.skipSharing)}</>}
-      >
-        {newIdeaId && (
-          <SharingModalContent
-            postType="idea"
-            postId={newIdeaId}
-            title={formatMessage(
-              getInputTermMessage(inputTerm, {
-                idea: messages.sharingModalTitle,
-                option: messages.optionSharingModalTitle,
-                project: messages.projectSharingModalTitle,
-                question: messages.questionSharingModalTitle,
-                issue: messages.issueSharingModalTitle,
-                contribution: messages.contributionSharingModalTitle,
-              })
-            )}
-            subtitle={formatMessage(messages.sharingModalSubtitle)}
-          />
-        )}
-      </Modal>
+      {typeof newIdeaId === 'string' && (
```

Comment:
I see. I rather see it as an explicit and informative check. :) Will leave this one for now.

Reply:
I could add a length check, but here it's not super relevant as an idea id, if it's a string, will not be an empty string. Otherwise there's a problem in a different place. (We don't do length checks for ids elsewhere either.)

Reply:
Ah, I only saw your comment after I merged. What do you mean? How does this lead to more checks in other places?

Reply:
It's not minor; it's good that you raised it. ;) My point was that the idea id, when it's a string, should never be an empty string. We make that assumption throughout the app, right? We often pass them as props, where we only check their type and not their length.  

If the idea id can be an empty string, that would point to a mistake in a different part of the app, so the problem may not surface when we check the length of the string and don't render the component. I'd rather have a crash of the app/request then.

There's certainly something to your point, though. We're making assumptions about certain values and are not strictly checking every aspect of them.